pkg/api/validate/admin.go

```diff
@@ -78,6 +78,7 @@ func (v *AdminAPIValidator) validateUpdateContainerService(cs, oldCs *api.OpenSh
 	old.Config.ImageVersion = cs.Config.ImageVersion
 
 	old.Config.Images = cs.Config.Images
+	old.Config.SecurityPatchPackages = cs.Config.SecurityPatchPackages
```

Comment:
will need to add suitable validation for SecurityPatchPackages

Reply:
I added an initial rudimentary check that the supplied rpm name doesn't end with `.rpm`. I also probably still need to check that the names match the format described here https://fedoraproject.org/wiki/Packaging:Naming?rd=Packaging:NamingGuidelines#Common_Character_Set_for_Package_Naming